library/mediawiki

```diff
@@ -2,12 +2,12 @@ Maintainers: David Barratt <dbarratt@wikimedia.org> (@davidbarratt),
              Kunal Mehta <legoktm@wikimedia.org> (@legoktm),
              addshore <addshorewiki@gmail.com> (@addshore)
 GitRepo: https://github.com/wikimedia/mediawiki-docker.git
-GitCommit: b5f46594d3229559a05090f3c89a469e874abd16
+GitCommit: 0a925a63be9b3b2c1f24779f6056743fbf814abe
 
 Tags: stable, latest, 1.29, 1.29.1
 Directory: stable
 
-Tags: legacy, 1.28, 1.28.2
+Tags: legacy
 Directory: legacy
 
 Tags: lts, 1.27, 1.27.3
```

Comment:
Why not just move the `legacy` tag down here for now instead of duplicating the `Dockerfile` verbatim?

Reply:
uhh... because in 1 month it will be different (i.e. a new legacy version will be introduced. Not sure if it would be better to get rid of the Dockerfile for 1 month or just leave it since we know it will be updated soon. Which would you prefer?

Reply:
If they're going to have identical content, then I'd prefer to see the tag move and have that other entry come back (and this tag move to it) when that happens.

Reply:
@tianon this is fixed now.